rest/model/user.go

```diff
@@ -200,22 +200,22 @@ func (n *APINotificationPreferences) ToService() (interface{}, error) {
 	spawnHostOutcome := utility.FromStringPtr(n.SpawnHostOutcome)
 	commitQueue := utility.FromStringPtr(n.CommitQueue)
 	if !user.IsValidSubscriptionPreference(buildbreak) {
-		return nil, errors.New("Build break preference is not a valid type")
+		return nil, errors.New("invalid subscription type build break")
```

Comment:
I think it would be more useful for these error messages to have the format `invalid build break subscription preference '%s', buildBreak`.